index.js

```diff
@@ -221,9 +221,13 @@ module.exports = {
       userPostTransformPlugins
     ).filter(Boolean);
 
-    options.presets = [
-      shouldRunPresetEnv && this._getPresetEnvPlugins(addonProvidedConfig),
-    ]
```

Comment:
Would it be easier to add `.filter(Boolean)` here instead?

Reply:
I think the condition is easier to read than `&&`

Reply:
peer pressure 😝